infoportal-client/src/features/Database/KoboTable/customization/useCustomSelectedHeader.tsx

```diff
@@ -15,15 +15,21 @@ export const useCustomSelectedHeader = (selectedIds: KoboAnswerId[]): ReactNode
     switch (ctx.form.id) {
       case KoboIndex.byName('shelter_cashForShelter').id: {
         return (
-          <SelectStatusBy
-            enum="ShelterCashStatus"
-            disabled={!ctx.canEdit}
-            sx={{maxWidth: 120}}
-            label={m.project}
-            onChange={_ => {
-              ctx.asyncUpdateTag.call({answerIds: selectedIds, value: _, key: 'status'})
-            }}
-          />
+          <>
+            <SelectStatusBy
+              enum="ShelterCashStatus"
+              disabled={!ctx.canEdit}
+              sx={{ maxWidth: 120, mr: 1 }}
+              label={m.status}
+              onChange={_ => {
+                ctx.asyncUpdateTag.call({ answerIds: selectedIds, value: _, key: 'status' });
+              }}
+            />
+            <IpDatepicker
+              label={m.paidOn}
+              onChange={_ => ctx.asyncUpdateTag.call({ answerIds: selectedIds, value: _, key: 'lastStatusUpdate' })}
```

Comment:
Just to harmonize ourself and avoid unnecessary conflict, when you reformat your code it format like this:
```all({ answerIds: selectedI``` and not like this
```all({answerIds: selectedI``` ?

If so I may have sent you the wrong intellij formatter. Let me know